### PR TITLE
Ignore .next from hot reload watcher

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

**Comment by rauchg** [comment]
Why don't we ignore dot-anything?

File: server/hot-reloader.js [hunk 1]
```diff
@@ -128,7 +128,8 @@ export default class HotReloader {
 
     const ignored = [
       /(^|[/\\])\../, // .dotfiles
```
Comment (arunoda) [inline server/hot-reloader.js]: @timneutkens does this(^^^) rule is also looking at `.next`?

Comment (timneutkens) [inline server/hot-reloader.js]: It does. I thought @rauchg closed this instead of merge 🤔

Comment (arunoda) [inline server/hot-reloader.js]: Cool. I'll revert.

**Pull request merged**